dataactbroker/handlers/account_handler.py

```diff
@@ -470,3 +470,57 @@ def list_user_emails():
 
     user_info = [{"id": user.user_id, "name": user.name, "email": user.email} for user in users]
     return JsonResponse.create(StatusCode.OK, {"users": user_info})
+
+
+def list_submission_users(d2_submission):
+    """ List user IDs and names that have submissions that the requesting user can see.
+
+        Arguments:
+            d2_submission: boolean indicating whether it is a DABS or FABS submission (True if FABS)
+
+        Returns:
+            A JsonResponse containing a list of users that have submissions that the requesting user can see
+    """
+
+    # print(sess.query(Submission).filter(Submission.user_id == 2).exists())
```

Comment:
Should this be logged? Or just simply removed?

Reply:
Oops, fixing